src/WebUI/Services/ServiceDetails.tsx

```diff
@@ -123,6 +126,10 @@ export class ServiceDetails extends BaseComponent<IServiceDetailsProperties, ISe
         this._servicesStore.removeListener(ServicesEvents.ServicePodsFetchedEvent, this._onPodsFetched);
     }
 
+    public componentDidMount(): void {
+        KubeFactory.telemetryService.scenarioEnd(Scenarios.ServiceDetails);
```

Comment:
we are loading pods asynchronously, how are we dealing with it? when you do refresh?